src/cocoa/toga_cocoa/widgets/webview.py

```diff
@@ -66,3 +68,78 @@ def evaluate(self, javascript):
         :type  javascript: ``str``
         """
         return self.native.stringByEvaluatingJavaScriptFromString(javascript)
+
+
+if "WKWebView" in vars():
```

Comment:
The pattern we've used elsewhere is to define `WKWebView = None` on an import failure, and then check if `WKWebView is None`.

We should also be using this as a guard for defining `WebKit1WebView` at all - if the platform has support for WKWebkit, there's no reason to define the old v1 web view widget.

Reply:
The concern here is that `WKWebView` is being imported through `from toga_cocoa.libs import *`, so there is no way to know in advance if WKWebView is presented inside `vars()`. 
For `WebKit1WebView`, since the next release of toga is `0.3`, so I think I probably should not break the API without any notice. But I am not familiar with the toga release cycle, if you think it is proper to change the API without deprecation, I can update the code to comply this.